glad/generator/rust/templates/impl.rs

```diff
@@ -1,5 +1,5 @@
 {% import 'template_utils.rs' as template_utils with context %}
-
+#![allow(non_snake_case)]
```

Comment:
From my tests this is not necessary, and if it was we should move it to the block where it is required (as with the other allows)